e2e/test/scenarios/admin/settings/sso/ldap.cy.spec.js

```diff
@@ -141,6 +158,44 @@ describe(
   },
 );
 
+describeEE("LDAP EE", { tags: "@external" }, () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    setTokenFeatures("all");
+  });
+
+  it("should allow user login on EE when LDAP is enabled", () => {
+    setupLdap();
+    cy.signOut();
+    cy.visit("/auth/login");
+    cy.findByLabelText("Username or email address").type("user01@example.org");
+    cy.findByLabelText("Password").type("123456");
+    cy.button("Sign in").click();
+    cy.findByTestId("main-navbar-root").within(() => {
+      cy.findByText("Home").should("exist");
+    });
+
+    cy.signOut();
+    cy.signInAsAdmin();
+
+    // Check that attributes are synced
+    cy.visit("/admin/people");
+    cy.get(".ContentTable").within(() => {
+      cy.findByText("Bar1 Bar1")
+        .closest("tr")
+        .within(() => {
+          cy.icon("ellipsis").click();
+        });
+    });
+    popover().within(() => {
+      cy.findByText("Edit user").click();
+    });
+    cy.get("input[value='uid']").should("exist");
+    cy.get("input[value='user01@example.org']").should("exist");
```

Comment:
Nit, and mostly just FYI
```suggestion
    cy.findByDisplayValue("uid").should("exist");
    cy.findByDisplayValue("user01@example.org").should("exist");
```